### PR TITLE
GSLUX-540: Handle WMTS layers

### PR DESCRIPTION
**Pull request by AlitaBernachot**
### JIRA issue

https://jira.camptocamp.com/browse/GSLUX-540

### Description

In theme / main / parcelles cadastrales we have a lot of WMTS, they are not handled yet.
Add this support in the openlayers service.

### CODE REVIEW

**Review by tkohr — APPROVED** [review]
Thanks for the quick migration of this feature @AlitaBernachot ! Works fine for internal layers and code LGTM. To be decided what we do with external WMTS...

File: src/composables/map/ol.composable.ts [hunk 1]
```diff
@@ -1,27 +1,73 @@
...
 import type { Layer, LayerId } from '@/stores/map.store.model'
+import { PROJECTION_DESTINATION, PROJECTION_SOURCE } from './map.composable'
+import { isHiDpi, stringToBoolean } from '@/services/utils'
+import { storageHelper } from '@/services/state-persistor/storage/storage.helper'
+import { SP_KEY_IPV6 } from '@/services/state-persistor/state-persistor.model'
 
 const proxyWmsUrl = 'https://map.geoportail.lu/ogcproxywms'
 export const remoteProxyWms = 'https://map.geoportail.lu/httpsproxy'
 
+const TILE_GRID_RESOLUTION = [
```
Comment (mki-c2c) [inline src/composables/map/ol.composable.ts:20]: would it be possible to move the constant definitions into a config file ?

Comment (AlitaBernachot) [reply src/composables/map/ol.composable.ts]: Yes, moved into wmts fixture file.

File: src/composables/map/ol.composable.ts [hunk 2]
```diff
@@ -32,11 +78,61 @@ function createWmsLayer(layer: Layer): ImageLayer<ImageWMS> {
...
+      'olcs.extent': getOlcsExtent(),
+      label: name,
+      id,
+    },
+  })
+
+  return olLayer
+}
+
+function getLayerWmtsUrl(layer: Layer, requestScheme = 'https') {
```
[the remark this answers is not in the document]
Comment (AlitaBernachot) [reply src/composables/map/ol.composable.ts:91]: As discussed together, this will be done in a separate ticket.

**Pull request merged**